boards/arm/adi_eval_adin1110ebz/doc/index.rst

```diff
@@ -0,0 +1,170 @@
+.. _adi_eval_adin1110ebz:
+
+ADI EVAL-ADIN1110EVB Evaluation board
+#####################################
+
+Overview
+********
+
+The EVAL-ADIN1110EBZ is a flexible platform enabling quick evaluation of the ADIN1110, robust,
+low power 10BASE-T1L MAC-PHY. It provides 10Mbit per second Single Pair Ethernet (SPE) connections
+with devices across 1.7km of cable.
+
+The evaluation board offers two modes of operation for maximum flexibility. Connected to a PC
+via USB port, the full set of ADIN1110 register settings and features such as link quality
+monitoring and diagnostics can be accessed over the USB using serial command interface.
+The board also provides an Arduino interface.
+
+Alternatively, the board can operate in stand-alone mode where it is configured by setting hardware
+configuration links and switches. On-board LEDs provide status indication.
+
+The SPI interface provides configuration and data access to the ADIN1110.
+
+A small prototyping area and test points are provided for experimentation with alternative cable
+connection topologies including isolation transformers and/or power coupling inductors.
+
```

Comment:
Sorry if I missed it, but it doesn't look like the image is included (along with the ADIN2111EBZ).

```suggestion
.. figure:: img/adi_eval_adin1110ebz.webp
   :align: center
   :alt: ADI EVAL-ADIN1110EVB

   ADI EVAL-ADIN1110EVB (Credit: ???)
```

Reply:
thansk, fixed